### PR TITLE
Hide Dropdowns on clicks outside of dropdowns

### PR DESCRIPTION
**Pull request by schwers**
https://reddit.atlassian.net/browse/MOBILEWEB-137

Hides Dropdowns when other elements on the page are clicked

:eyeglasses: @ajacksified

### CODE REVIEW

File: assets/js/client.es6.js [hunk 1]
```diff
@@ -452,8 +462,20 @@ function initialize(bindLinks) {
     }
   });
 
-  window.addEventListener('scroll', throttle(function() {
+  function closeDropdowns(e) {
```
Comment (ajacksified) [inline assets/js/client.es6.js]: `e` doesn't appear to be used anywhere (or passed in)

File: src/constants.es6.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ export default {
   SIDE_NAV_TOGGLE: 'sideNavToggle',
   TOP_NAV_HAMBURGER_CLICK: 'topNavHamburgerClick',
   VOTE: 'vote',
+  DROPDOWN_CSS_CLASS: 'Dropdown',
```
Comment (ajacksified) [inline src/constants.es6.js]: someday, we can pass this into LESS as a variable :wishful_thinking:

**Comment by ajacksified** [comment]
Just a small nit. :nail_care:

**Comment by schwers** [comment]
Whoops, I'll clean that up

**Comment by schwers** [comment]
:haircut:

**Comment by ajacksified** [comment]
:fish:

**Pull request merged**